### ISSUE TITLE
Images sent are downsized (they shouldn' be, or not too much).

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/OpenFoodFacts-androidApp#447]
https://de.openfoodfacts.org/images/products/730/040/012/2702/1.jpg
https://de.openfoodfacts.org/images/products/730/040/012/2702/2.jpg
```mikeh Images uploaded with the new app are still extreamly downsized```

**Comment by itchix** [comment]
We compress images when the app send the data : https://github.com/openfoodfacts/OpenFoodFacts-androidApp/blob/master/app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java#L114

**Comment by teolemon** [comment]
We can fix a higher minimum and let the user decide for very high res ?

**Comment by teolemon** [comment]
Also, it would be great to try to send images as soon as they are selected, and not wait for Send/Save

**Comment by asmoth360** [comment]
Agreed to upload larger images, it's cropped then to view only the information we want, so it needs to be reasonably big. Maybe an option to send original pictures when connected in Wifi ?

**Comment by fuddl** [comment]
what was wrong with how the old app did it? (upload the original file)

Only scheduled when wifi is available
